backend/src/db/schemas/compliance.ts

```diff
@@ -215,29 +215,6 @@ const ComplianceReportSchema = new mongoose.Schema({
   }
 });
 
-ComplianceDetailSchema.pre('save', function (next) {
```

Comment:
This is unnecessary, proper validation is added here: https://github.com/GovStackWorkingGroup/testing-webapp/pull/124
I delete this to ensure there wont be any issues with this outdated form validation.